Fixes script samples options with v6 release

milanholemans opened the pull request:
Closes #4441

Fixes all sample scripts to be in line with our latest major `v6` release.

Jwaegebaert commented:
Awesome work @milanholemans 💪

waldekmastykarz commented:
Great job @milanholemans! Cool to see these samples aligned with out latest version! 👏

milanholemans commented:
Putting is back in draft because I forgot to update the CLI version in the sample.json files.

Adam-it commented on docs/docs/sample-scripts/spo/list-all-files-with-missing-required-metadata/index.md:
````diff
@@ -14,13 +14,13 @@ The following script iterates through all files in all the document libraries of
     ```powershell
     $siteUrl = "https://<TENANT-NAME>.sharepoint.com/sites/<YOUR-SITE>"
-    $allLists = m365 spo list list --webUrl $($siteURL) --output json | ConvertFrom-Json
+    $allLists = m365 spo list list --webUrl "$($siteURL)" --output json | ConvertFrom-Json
````
TBH I don't think we need the `$()` here

```suggestion
    $allLists = m365 spo list list --webUrl "$siteURL" --output json | ConvertFrom-Json
```

milanholemans replied:
Yes correct :)

Adam-it commented on docs/docs/sample-scripts/spo/list-all-files-with-missing-required-metadata/index.md:
```diff
@@ -14,13 +14,13 @@ The following script iterates through all files in all the document libraries of
 
     foreach ($lib in $allLibs) {
-        $allRequiredFields = m365 spo field list --webUrl $($siteURL) --listId $($lib.Id) --query "[?Required == ``true``]" --output json | ConvertFrom-Json
+        $allRequiredFields = m365 spo field list --webUrl "$($siteURL)" --listId $($lib.Id) --query "[?Required == ``true``]" --output json | ConvertFrom-Json
```
same comment I think it will still work without the `$()`

```suggestion
        $allRequiredFields = m365 spo field list --webUrl "$siteURL" --listId $($lib.Id) --query "[?Required == ``true``]" --output json | ConvertFrom-Json
```

Adam-it commented on docs/docs/sample-scripts/spo/list-all-files-with-missing-required-metadata/index.md:
```diff
@@ -36,7 +36,7 @@ The following script iterates through all files in all the document libraries of
         }
 
-        $allItems = m365 spo listitem list --webUrl $($siteURL) --listId $($lib.Id) --fields $($allRequiredFieldsInternalName -join ",") --output json | ConvertFrom-Json
+        $allItems = m365 spo listitem list --webUrl "$($siteURL)" --listId $($lib.Id) --fields $($allRequiredFieldsInternalName -join ",") --output json | ConvertFrom-Json
```
```suggestion
        $allItems = m365 spo listitem list --webUrl "$siteURL" --listId $($lib.Id) --fields $($allRequiredFieldsInternalName -join ",") --output json | ConvertFrom-Json
```

Adam-it commented on docs/docs/sample-scripts/spo/list-all-list-folders-itemcount/index.md:
```diff
@@ -30,7 +30,7 @@ List all Lists, the folders and sub folders in a given site, and output the item
 
       foreach ($folder in $folders) {
-        $folderStats = m365 spo folder get -u $webUrl --folderUrl $folder.ServerRelativeUrl -o json | ConvertFrom-Json
+        $folderStats = m365 spo folder get -u $webUrl --url $folder.ServerRelativeUrl -o json | ConvertFrom-Json
```
I know it will work but it just seems kinda weird to mix short with long option naming 😅. Could we align 😉?

```suggestion
        $folderStats = m365 spo folder get -u $webUrl -f $folder.ServerRelativeUrl -o json | ConvertFrom-Json
```

Adam-it commented on docs/docs/sample-scripts/teams/export-all-channels-teams/index.md:
```diff
@@ -1,4 +1,4 @@
----
+4---
```
I think it was added by mistake 🙂

```suggestion
---
```

Adam-it reviewed (changes requested):
@milanholemans I am really amazed by the awesome work you have done 👍 
I added some cosmetic comments, please give them a double check 🙂
you rock 🤩

milanholemans commented:
Hi @Adam-it, thank you for the feedback! Updated my PR with your comments.
I also updated the version numbers to `v6.3` since this is already around the corner and because I made some enhancements to [flow run list](https://pnp.github.io/cli-microsoft365/cmd/flow/run/run-list) commands to use the `status`, `triggerStartTime` and `triggerEndTime` which is included in our `v6.3` version. I hope these enhancements are ok for you.

Adam-it commented:
@milanholemans thanks for the quick turnaround 👍. I will review it ASAP.  I think is ok to already update to `6.3` but since it is not released (yet) and I plan to do some merges today I will leave this one for next time (some time next week 😉)
BTW again awesome work. Your rock 🤩

Adam-it reviewed (approved):
awesome work 👍

Adam-it commented:
ready to merge 🚀

Adam-it commented:
merged manually. Awesome work 👍. You rock 🤩
